geo-types/src/point.rs

```diff
@@ -409,6 +414,79 @@ where
     }
 }
 
+#[cfg(test)]
+impl<T> RelativeEq for Point<T>
+where
+    T: CoordinateType + Float,
+{
+    // type Epsilon = T;
+
+    #[inline]
+    fn default_max_relative() -> Self::Epsilon {
+        T::epsilon()
+    }
+
+    #[inline]
+    fn relative_eq(
+        &self,
+        other: &Point<T>,
+        epsilon: Self::Epsilon,
+        max_relative: Self::Epsilon,
+    ) -> bool {
+        // Handle same infinities.
```

Comment:
I was initially surprised that we weren't simply delegating to the `approx::relative_eq`, but I think I understand why.

It might be helpful for future readers if there were a comment that explained this - something like:

```
// this implementation is based on the [approx crates `relative_eq`](some link) 
// implementation, but rather than comparing the difference
// between two scalars, we compare the euclidean distance between two 
// points
```
(feel free to improve)